config-templates/complex-tiflash.yaml

```
@@ -124,6 +124,10 @@ tiflash_servers:
     #   # storage.latest.capacity: [ 161061273600 ]
     # learner_config:
     #   log-level: "info"
+    #   server.labels:
+    #     zone: "z2"
+    #     dc: "dc2"
+    #     host: "host2"
```

Maybe keep the same format as tikv does in line 95?
```suggestion
    #   server.labels: { zone: "zone1", dc: "dc1", host: "host1" }
```

---

> Ansible doesn’t really use these too much.
> https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

Multiple lines representations are more recommended in yaml.

---

Maybe align the style of tikv to tiflash? Or it maybe confusing that two style in the same yaml file.